test/apiv2/20-containers.at

```diff
@@ -374,8 +378,13 @@ t DELETE containers/$cid?v=true 204
 t POST containers/create Image=$IMAGE HostConfig='{"NetworkMode":"default"}' 201 \
   .Id~[0-9a-f]\\{64\\}
 cid=$(jq -r '.Id' <<<"$output")
+if root; then
+    network_type=bridge
+else
+    network_type=slirp4netns
+fi
 t GET containers/$cid/json 200 \
-  .HostConfig.NetworkMode="bridge"
+  .HostConfig.NetworkMode="$network_type"
```

Comment:
This looks like a bug, containers created via compat api should always have bridge as default even as rootless, see 970c8d4724

Given that there are test differences for rootless we should add a job to run this rootless in CI to prevent such regression in the future.